components/02-molecules/pull-quote/pull-quote.twig

```diff
@@ -0,0 +1,35 @@
+{#
+ # Available Props:
+ # - pull_quote__style: bar-left, bar-right, or quote-left
+ #
+ # Available Variables:
+ # - pull_quote__quote (required)
+ # - pull_quote__attribution
+ #}
+
+{% set pull_quote__base_class = 'pull-quote' %}
+{% set pull_quote__style = pull_quote__style|default('bar-left') %}
+
+{# If pull_quote__attributes is not defined, set it to an empty object by default #}
+{% set pull_quote__attributes = pull_quote__attributes|default({}) %}
+
+{% set pull_quote__attributes = pull_quote__attributes|merge({
+  'data-component-width': 'content',
+  'data-pull-quote-style': pull_quote__style,
+  class: bem(pull_quote__base_class),
+}) %}
+
+<aside {{ add_attributes(pull_quote__attributes) }}>
+  <div {{ bem('inner', [], pull_quote__base_class) }}>
+    <figure {{ bem('figure', [], pull_quote__base_class) }}>
+      {% include "@atoms/text/text/02-blockquote.twig" with {
+        blockquote_base_class: 'quote',
+        blockquote_content: pull_quote__quote,
+        blockquote_blockname: 'pull-quote',
```

Comment:
I'd just reassign this to the base class so you're not writing `pull-quote` twice. `blockquote_blockname: pull_quote__base_class,`